easybuild/easyconfigs/a/Amber/Amber-16-CrayGNU-17.08-cuda-8.0-pat-6.5.1.eb

```diff
@@ -0,0 +1,52 @@
+# @author: karakasv gppezzi
+# @author: victor holanda rusu
+
+easyblock = 'MakeCp'
+
+name = 'Amber'
+version = '16'
+cudaversion = '8.0'
+patversion = '6.5.1'
+versionsuffix = '-cuda-%s-pat-%s' % (cudaversion, patversion)
+
+homepage = 'http://ambermd.org/'
+description = 'Assisted Model Building with Energy Refinement'
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+toolchainopts = { 'verbose' : False }
+
+sources = ['/apps/common/UES/easybuild/sources/%(nameletterlower)s/%(name)s/%(name)s16.tar.bz2',
+           '/apps/common/UES/easybuild/sources/%(nameletterlower)s/%(name)s/AmberTools16.tar.bz2']
+
+dependencies = [ ('bzip2', '1.0.6'),
+                 ('zlib', '1.2.11') ]
+
+builddependencies = [
+    ('cudatoolkit/8.0.61_2.4.3-6.0.4.0_3.1__gb475d12', EXTERNAL_MODULE),
+    ('cray-hdf5/1.10.0.3', EXTERNAL_MODULE),
+    ('cray-netcdf/4.4.1.1.3', EXTERNAL_MODULE),
+    ('flex', '2.6.4'),
+    ('perftools-lite-gpu', EXTERNAL_MODULE),
+]
+
+buildininstalldir = True
+
+prebuildopts = 'cd .. && mv amber%s/* . && ' % version
+prebuildopts += 'export AMBERHOME=%(builddir)s; export CUDA_HOME=$CUDATOOLKIT_HOME;'
+prebuildopts += './configure -mpi -cuda -noX11 -crayxt5 --skip-python --with-netcdf $EBROOTNETCDF gnu <<< Y;'
```

Comment:
What means this ? <<<Y

Reply:
It means, yes to the question it asks in the command line.
The question is "do you want to apply the update patches in Amber?"
And we say yes.